### PR TITLE
fixed create method to use proper scope

### LINKED ISSUE #9913: [Bug]: couchbase vector taking default scope as color  while creating the tables even if correct scope is provided
### Short description of current behavior

#9906
couchbase vector taking default scope as color  while creating the tables

### Video or screenshots

table creation command fails
{E12D08B0-027E-4741-9A6C-105F97D561CF}

{45CED87E-887C-48AC-896F-5D1E7B67E52B}

### Expected behavior

_No response_

### How to reproduce the error

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Sekhar-Kumar-Dash**
## Description

couchbase vector was taking default scope as color while creating the tables
this PR solves that issue

Fixes #9913

## Type of change

(Please delete options that are not relevant)

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.
{C9F4CEAD-5B1E-47B3-B2A7-B2605C02C1E7}

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [x] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by mindsdbadmin** [comment]
<br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb/blob/main/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request. </sub><sub>Posted by the **CLA Assistant Lite bot**.</sub>

**Comment by Sekhar-Kumar-Dash** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by ZoranPandovski** [comment]
recheck

File: mindsdb/integrations/handlers/couchbasevector_handler/couchbasevector_handler.py [hunk 1]
```diff
@@ -366,10 +366,12 @@ def create_table(self, table_name: str, if_not_exists=True):
         """
         cluster = self.connect()
         bucket = cluster.bucket(self.bucket_name)
-        scope = bucket.scope(self.scope)
-        _ = scope.collection(table_name)
+        scope = bucket.scope(self.scope)  
         try:
-            bucket.collections().create_collection(scope_name="color", collection_name=table_name)
+            bucket.collections().create_collection(
+                scope_name=self.scope,
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/couchbasevector_handler/couchbasevector_handler.py:372]: Should't this be the `scope` ?

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/handlers/couchbasevector_handler/couchbasevector_handler.py (hunk 1 above)
Comment (Sekhar-Kumar-Dash) [reply mindsdb/integrations/handlers/couchbasevector_handler/couchbasevector_handler.py]: fixed it @ZoranPandovski

Comment (ZoranPandovski) [reply mindsdb/integrations/handlers/couchbasevector_handler/couchbasevector_handler.py]: Sorry this was a question about the value for the `scope_name`. Do we need the self.scope or scope object?

Comment (Sekhar-Kumar-Dash) [reply mindsdb/integrations/handlers/couchbasevector_handler/couchbasevector_handler.py:372]: i think we need self.scope 
this sets self.scope as either the scope name from connection data or default

**Review by ZoranPandovski — APPROVED** [review]

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
Please fix the flak8 warnings https://github.com/mindsdb/mindsdb/actions/runs/11478236155/job/31942034752?pr=9914#step:6:115

File: mindsdb/integrations/handlers/couchbasevector_handler/couchbasevector_handler.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import ast
+]import ast
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/couchbasevector_handler/couchbasevector_handler.py:1]: Please remove the ]

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
Can you revert all changes and just keep your fix. If you try to lint everything it will be hard to review this

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**